backend/app/main.py

```
@@ -0,0 +1,30 @@
+from typing import Union
+from fastapi import FastAPI
+
+from database import get_db, Base, engine
```

Ruff (0.8.2)

`database.get_db` imported but unused

Remove unused import: `database.get_db`

(F401)